homeassistant/components/sensibo/quality_scale.yaml

```diff
@@ -0,0 +1,77 @@
+rules:
```

Comment:
```py
    SensiboDeviceBinarySensorEntityDescription(
        key="room_occupied",
        translation_key="room_occupied",
        device_class=BinarySensorDeviceClass.MOTION,
        value_fn=lambda data: data.room_occupied,
    ),
```
This sounds like PRESENCE tbf

Reply:
Presence is home/away. This is motion for a room.